lib/Conversation/CommentForm/styles.scss

```diff
@@ -42,6 +42,21 @@ $comment-form-padding: $layout-spacing-base/2;
 /* ==========================================================================
    State Styles
    ========================================================================== */
-.has-value .comment-form__actions--submit {
-  display: inline-block;
-}
\ No newline at end of file
+   .new-conversation .is-submitted .button--secondary {
```

Comment:
Is this styling for a disabled state? I wonder if this should be abstracted away from the submit form component and into the Button component, what do you think?

We could then update the type of button when required.